src/emulator/functionsEmulator.ts

```diff
@@ -115,6 +115,13 @@ export class FunctionsEmulator implements EmulatorInstance {
     // TODO: Would prefer not to have static state but here we are!
     EmulatorLogger.verbosity = this.args.quiet ? Verbosity.QUIET : Verbosity.DEBUG;
 
+    // When debugging is enabled, the "timeout" feature needs to be disabled so that
+    // functions don't timeout while a breakpoint is active.
+    if (this.args.debugPort) {
```

Comment:
lgtm, can't imagine anything going wrong

Reply:
I hope to use a screenshot of this comment in a P0 bug one day.